lib/git.js

```diff
@@ -52,14 +53,40 @@ module.exports = async function (context, token, commits, target, logger) {
           '--abort',
         ])
 
-        await git.raw([
-          'cherry-pick',
-          commits[i],
-          '--strategy-option',
-          'ours'
-        ], (err, result) => {
-          logger.info(err, result)
-        })
+        let originalCommitMessage = undefined
+        try {
+          const commitLog = await git.log({
+            from: commits[i],
+            to: commits[i],
```

Comment:
```suggestion
            to: `${commits[0]}~1`,
```

The syntax `COMMIT_HASH~1` means 1 commit after COMMIT_HASH (which itself is not included). Now, a single commit is returned (`commits['i']`).

Reply:
Otherwise, `git.log()` will just return an empty log: `ListLogSummary { all: [], latest: null, total: 0 }`